bin/check_samplesheet.py

```diff
@@ -67,10 +67,8 @@ def check_samplesheet(file_in, file_out):
 
             ## Check sample name entries
             sample, fastq_1, fastq_2, strandedness = lspl[:len(HEADER)]
-            if sample:
-                if sample.find(" ") != -1:
-                    print_error("Sample entry contains spaces!", "Line", line)
-            else:
+            sample = sample.replace(' ', '_')
```

Comment:
Although unlikely, this could, in theory, lead to name clashes.

Reply:
So you think we should keep the old behaviour and fail? I thought it would just be less hassle for users but you're right.

Reply:
It's probably ok. How about writing out a warning message just in case, something like: 
```
NOTE: Spaces in sample <xxx> have been replaced by underscores
```

Reply:
Added https://github.com/nf-core/rnaseq/pull/660/commits/3eb2bac959b5987ae777325f232ee6b4c1a19791